src/patternfly/components/Tabs/examples/tabs-secondary.hbs

```diff
@@ -0,0 +1,13 @@
+<div class="pf-c-tabs-container">
```

Comment:
I agree, `pf-c-tabs` is now the container and buttons are now within lists. This required a lot of restructuring of the handlebars files, so let me know if it looks okay.

Reply:
FYI, the current implementation has some positioning issues. I'm not quite sure the buttons can be stickied/fixed to the left and right while being within `.pf-c-tabs__list`, but I'll keep trying.

Reply:
Thanks to @mcoker and @mattnolting I think we have a solution.

Tabs never need a container, whether or not secondary tabs are present. Instead, in the specific case where secondary tabs are directly adjacent to primary tabs, we use an adjacent sibling selector to style those secondary tabs appropriately (adding top border, subtracting margin to prevent stacked borders, setting `display:flex;` to move secondary tabs to a new line).

This seems to meet the design requirements. One side effect of setting secondary tabs to be full width without a container is that the right-most scroll button is positioned way off to the right. Since scroll buttons are conditionally activated with JavaScript, I'm not too concerned about that.

<img width="812" alt="secondary-tabs-right-scroll-button" src="https://user-images.githubusercontent.com/9122899/48490591-c13efd00-e7f3-11e8-8a28-d44f07eaf015.png">

Reply:
I'm not sure we can rely on secondary tabs being adjacent siblings of primary tabs.

When I select a primary tab, the tab panel associated with that selected primary tab becomes visible. Wouldn't the secondary tabs live in that tab panel?

Reply:
@jgiardino We only rely on secondary tabs (or the tab panel that contains them) being directly adjacent to the primary tabs if the styling in the example is what the dev is looking for (with primary tabs on top and secondary directly below them, which is probably pretty common, and the DOM should reflect that positioning).

Secondary tabs that are revealed by primary tabs could be somewhere completely separate in the UI, which isn't reflected in the design doc but is something that some products do (@mattnolting can attest). In that case our Primary + Adjacent Secondary styling won't apply.

Should the markup of our secondary tabs example be updated to have a container that has `role="tabpanel"`, or maybe putting that `role` directly onto `.pf-m-tabs-secondary` would work?

Reply:
I tweaked this so that the secondary tab component can be buried within a div and still receive the right styling, so long as that containing div has `.pf-m-tabs-secondary` and is directly adjacent to the primary tabs. React devs feel confident that they won't need containing divs, but it's there if needed.

The `.pf-m-tabs-secondary` modifier is still a bit weird because it changes the component's margins, but only in a highly-specific context that's "conceptually within the rules" even though it technically breaks them. The component itself doesn't have margins and can be dropped anywhere and work as usual, which we seem to be okay with.